e2e_test/streaming/on_conflict.slt

```diff
@@ -0,0 +1,47 @@
+statement ok
+create table t1 (v1 int, v2 int, v3 int, primary key(v1)) on conflict ignore;
+
+statement ok
+insert into t1 values (1,4,2), (2,3,3);
```

Comment:
please also add an conflict case in the same DML to make them in the same chunk.